Dockerfile.client-server-cg.rh

```diff
@@ -1,37 +1,7 @@
-#
-# Note about source containers:
-# Due to the way this image is built, the source container for the image itself
-# does not include the full set of sources for the cli tool binaries included in
-# the image. The complete sources for each cli tool are available in the source
-# containers for the images that the binaries are extracted from.
-#
-# You can find the source containers for each of the cli tools in the following
-# locations:
-#
-# Cosign
-#   https://catalog.redhat.com/software/containers/detail/652971cd584d77e9d922883b
-#   docker://registry.redhat.io/rhtas-tech-preview/cosign-rhel9
-#
-# Gitsign
-#   https://catalog.redhat.com/software/containers/detail/652974c4f28b8157c60653c9
-#   docker://registry.redhat.io/rhtas-tech-preview/gitsign-rhel9
-##
+# Provides the Trusted Artifact Signer CLI binaries, cosign and gitsign
 
-# An image with oc so we can run `oc image extract` to download the binaries
-FROM registry.redhat.io/openshift4/ose-tools-rhel8:v4.11.0-202401122348.p0.gbf40a6c.assembly.stream AS downloads
-
-ARG COSIGN_IMAGE=quay.io/redhat-user-workloads/rhtas-tenant/cli/cosign@sha256:8cc2fcf240357c9ace8fea16da0c1b27fa91a03c53e16cd909ce9aa20d01a1f9
-ARG GITSIGN_IMAGE=quay.io/redhat-user-workloads/rhtas-tenant/cli/gitsign@sha256:4c67f8eb3f3156b47e31ebffeae950114943e560e0396cd6320dc7de5bdc255d
-
-RUN mkdir -p /downloads/cosign && \
-    mkdir -p /downloads/gitsign
-
-RUN oc image extract ${COSIGN_IMAGE}  --path /usr/local/bin/cosign*:/downloads/cosign && \
-    oc image extract ${GITSIGN_IMAGE} --path /usr/local/bin/gitsign_cli_*:/downloads/gitsign
-
-# So we have a consistently named and gzipped file
-RUN gzip /downloads/cosign/cosign && \
-    mv /downloads/cosign/cosign.gz /downloads/cosign/cosign-linux-amd64.gz
+FROM quay.io/redhat-user-workloads/rhtas-tenant/cli/cosign@sha256:4d47a96d44efc1dba028646849e131a5da9e45a5dfc5d08316249c6b45d0fead AS cosign
```

Comment:
Hold this PR until the image here can be updated from a snapshot after 5:45pm GMT on March 19.